### DISCUSSION TITLE
How would I go about backing up a system to my NAS without ZFS on the NAS trying to mount any of the datasets?

### DISCUSSION THREAD

**Discussion by UntouchedWagons** [zbm-dev/zfsbootmenu#629] [Questions]
For example the dataset that would normally be mounted at `/home` would get replicated over to my NAS then would get mounted at the same mount point on my NAS which I don't want. What are my options?

**Comment by ahesford** [comment]
There are plenty of tools for ZFS replication, and if they are half serious, they will set `canmount=off` on the remote host. This topic is well beyond the scope of a ZFSBootMenu discussion.
